Project_Chronos/src/objectFiles/Meeting.java

```diff
@@ -187,14 +189,6 @@ public int getNumAvailableUsers(int row, int col) {
 		return curr_ava.getNumAvailableUsers();
 	}
 	
-
-	public int getNumDays() {
-		return numDays;
-	}
-	
-	public int getNumHoursPerDay() {
-		return numHoursPerDay;
-	}
```

Comment:
I know this can be accessed by getting the length and width of the Availability array, but I think we should keep it for clarity's sake. Also, the constructor takes these parameters in so might as well have the getters.

Reply:
They are duplicated getters because I did not update my local branch on time. We still have this getter in the whole file.

Reply:
Ohh got it, thanks for letting me know.